configs/forseti_conf.yaml.in

```diff
@@ -55,6 +55,9 @@ inventory:
     # Valid values are: debug, info, warning, error
     loglevel: info
 
+    # Number of days to retain inventory data, 0 to keep forever
+    retention_days: 30
```

Comment:
As we discussed in the issue, we would like this to default to no deleting data.

Also, using ```0```, seems easy to interpret in the opposite way, i.e. retain nothing, i.e. delete everything.
So, how about using ```-1``` to designate keep forever?

So, possibly we can comment:
-1 to keep forever
0 to delete everything